### PR TITLE
Add some more credential tests to local lambda python integ tests

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by kiiadi — APPROVED** [review]

File: jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLambdaRunConfigurationIntegrationTest.kt [hunk 1]
```diff
@@ -189,6 +190,34 @@ class PythonLocalLambdaRunConfigurationIntegrationTest(private val runtime: Runt
...
+            input = "\"Hello World\"",
+            credentialsProviderId = mockSessionId
+        )
+        assertThat(runConfiguration).isNotNull
+
+        val executeLambda = executeRunConfiguration(runConfiguration)
+
+        assertThat(executeLambda.exitCode).isEqualTo(0)
+        assertThat(jsonToMap(executeLambda.stdout))
+            .containsEntry("AWS_ACCESS_KEY_ID", mockCreds.accessKeyId())
```
Comment (hunterwerlla) [inline jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLambdaRunConfigurationIntegrationTest.kt:218]: it's comparing `Access` to `access` :(

**Review by hunterwerlla — APPROVED** [review]

File: jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLambdaRunConfigurationIntegrationTest.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLambdaRunConfigurationIntegrationTest.kt:218]: D'oh!

**Pull request merged**